src/common/Utils/useValueTracker.ts

```diff
@@ -5,7 +5,7 @@ export const useValueTracker = <T = any>(
   defaultValue?: T | undefined,
 ): [T | string, (value: T) => void] => {
   const initialValue = useMemo(() => {
-    return defaultValue ?? value ?? "";
+    return (value || defaultValue) ?? "";
```

Comment:
is this value tracker ever used for booleans `false` or `0`, if so this might break expected behaviour.